### DISCUSSION TITLE
Проблемы с регистрацией нового пользователя

### DISCUSSION THREAD

**Discussion by rkfsociety** [plan-player-analytics/Plan#2651] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/40957202/194409341-b14ce17b-323c-4885-9fb4-7e46a898f978.png)

Есть какие то предположения, может кто то уже сталкивался... Использую встроенный SSL в Plan.jar

**Comment by AuroraLS3** [comment, marked as answer]
Fixed in newer version
